pkg/modules/reverseproxy.go

```
@@ -132,6 +144,21 @@ func ReverseProxyServer(ctx context.Context, isDeleting bool, op utils.OperatorC
 
 	for _, ctrlObj := range deployObjects {
 		log.Infof("Object: %v -----\n", ctrlObj)
+		if ctrlObj.GetName() == RevProxyServiceName && ctrlObj.GetObjectKind().GroupVersionKind().Kind == "Deployment" {
+			dp := ctrlObj.(*appsv1.Deployment)
+
+			secretSupported, _ := utils.MinVersionCheck(drivers.PowerMaxMountCredentialMinVersion, cr.Spec.Driver.ConfigVersion)
+			if secretSupported {
+				if drivers.UseReverseProxySecret(&cr) {
+					secretName := cr.Spec.Driver.AuthSecret
+					deploymentSetReverseProxySecretMounts(dp, secretName)
+				} else {
+					revProxyModule, _, _ := getRevproxyApplyCR(cr, op)
```

Ignoring error values.